### ISSUE TITLE
[BUG] Typo in fullchem.eqn in KPP/fullchem

### ISSUE THREAD

**Issue by viral211** [geoschem/geos-chem#841]
There is a typo in the following reactions in fullchem.eqn:
https://github.com/geoschem/geos-chem/blob/d7baeb8fe51a66914408e9d1f655dd062bd5d4df/KPP/fullchem/fullchem.eqn#L1358-L1360
The product in these reactions should be HNO2, instead of HNO3 (cf. Table 2, Reaction R7 in [Wang et al. 2019](https://acp.copernicus.org/articles/19/3981/2019/acp-19-3981-2019.pdf)). Xuan Wang confirms and says that the fix should have only a minor effect.

**Comment by yantosca** [comment]
This is now fixed with commit 14dca24963dacdfd43cd41491786c68e60090a9c.

**Comment by msulprizio** [comment]
This update was tested with 1-month internal benchmark `13.3.0-alpha.6`. Results may be viewed at http://ftp.as.harvard.edu/gcgrid/geos-chem/validation/InternalBenchmarks/GCC_13.3.0-alpha.6/. The OH metrics are included below for quick review:

```
###############################################################################
### OH Metrics
### Ref = GCC_13.3.0-alpha.5; Dev = GCC_13.3.0-alpha.6
###############################################################################

------------------------------------------------------------
Global mass-weighted OH concentration [10^5 molec cm^-3]
------------------------------------------------------------
Ref      : 11.80156821346
Dev      : 11.80178350713
Abs diff :  0.00021529366
%   diff :  0.001824

------------------------------------------------------------
CH3CCl3 (aka MCF) lifetime w/r/t tropospheric OH [years]
------------------------------------------------------------
Ref      :  5.240901
Dev      :  5.240778
Abs diff : -0.000123
%   diff : -0.002342

------------------------------------------------------------
CH4 lifetime w/r/t tropospheric OH [years]
------------------------------------------------------------
Ref      :  8.831226
Dev      :  8.831014
Abs diff : -0.000212
%   diff : -0.002398
```
